main/test/ca/uwaterloo/flix/language/phase/TestRedundancy.scala

```diff
@@ -1278,4 +1278,47 @@ class TestRedundancy extends FunSuite with TestUtils {
     val result = compile(input, Options.TestWithLibNix)
     expectError[RedundancyError.RedundantUpcast](result)
   }
+
+  test("TestParYield.01") {
```

Comment:
Let's also test for unused var and shadowed var where the shadowing is inside a fragment.